backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala

```diff
@@ -156,22 +162,28 @@ trait StandardAsyncExecutionActor extends AsyncBackendJobExecutionActor with Sta
     * @param globFiles The globs.
     * @return The shell scripting.
     */
-  def globManipulations(globFiles: Traversable[WomGlobFile]): String = globFiles map globManipulation mkString "\n"
+  def globScripts(globFiles: Traversable[WomGlobFile]): String =
+    globFiles map globScript mkString "\n"
 
   /**
     * Returns the shell scripting for hard linking a glob results using ln.
     *
     * @param globFile The glob.
     * @return The shell scripting.
     */
-  def globManipulation(globFile: WomGlobFile): String = {
+  def globScript(globFile: WomGlobFile): String = {
     val parentDirectory = globParentDirectory(globFile)
-    val globDir = backendEngineFunctions.globName(globFile.value)
+    val globDir = GlobFunctions.globName(globFile.value)
```

Comment:
Just to make sure, there was no backend-specific code in any `globName` function override?

Reply:
Confirming there was no backend-specific code.